ofrak_core/test_ofrak/components/test_entropy_component.py

```diff
@@ -1,13 +1,11 @@
-import math
 import os.path
-from typing import List
 
 import pytest
 from ofrak.core.entropy import DataSummaryAnalyzer, DataSummary
 
 from ofrak import OFRAKContext
 import test_ofrak.components
-
+from ofrak.core.entropy.reference_entropy import entropy_func
```

Comment:
See comment above about testing both implementations.

We should test the functions themselves, in addition to the overall analyzer.